### PR TITLE
#13 feature: xbox input

### PR DESCRIPTION
**Pull request by Trebua**
**Det er Simen som har lagt ned arbeidet 🥇** 

Features
* Wrapper component for using external react-gamepad-component
* Sending joystick and button changes/clicks to ipcMain

Note:
* There is a little bug causing errors in the console, but it works for now.

Bug:
image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (eirikdahlen) [inline .vscode/settings.json]: This file should not be included in the PR?

Comment (eirikdahlen) [reply .vscode/settings.json]: Removed

File: public/electron.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ function createWindows() {
     y: 0,
     webPreferences: {
       nodeIntegration: true,
+      preload: __dirname + '/preload.js',
```
Comment (greenbech) [inline public/electron.js]: I think it is better pratice to use the `path` library to join file path because paths are different on Windows and UNIX-systems and suddently the code won’t run on my Mac. You can see here https://github.com/sindresorhus/electron-boilerplate/blob/bd56521469494cbb9741cc6e930229e0dcf6ff6d/index.js#L51 for an example
